channels/vn.m3u

```diff
@@ -29,3 +29,75 @@ http://thtg.vn:8001/thtg.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",LS TV
+https://livefta.malimarcdn.com/ftaedge00/laostv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Look Thoong TV
+https://livefta.malimarcdn.com/ftaedge00/lookthoongtv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Uniquely Thai
+https://livefta.malimarcdn.com/ftaedge00/uniquely.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vanphenh TV
+https://livefta.malimarcdn.com/ftaedge00/vanphenhtv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Sieng Khaen Lao TV
+https://livefta.malimarcdn.com/ftaedge00/vlctemple.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Lao Champa TV 1
+https://livefta.malimarcdn.com/ftaedge00/laochampa1.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Lao Champa TV 2
+https://livefta.malimarcdn.com/ftaedge00/laochampa2.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Lao Champa TV 3
+https://livefta.malimarcdn.com/ftaedge00/laochampa3.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",ISTV
+https://livefta.malimarcdn.com/ftaedge00/ichannel.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",NAT TV
+https://livefta.malimarcdn.com/ftaedge00/nat.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Lao SV TV
+https://livefta.malimarcdn.com/ftaedge00/laosvtv.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vajtswv Txojlus
+https://livefta.malimarcdn.com/ftaedge00/vajtswv.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",HmongUSA TV
+https://livefta.malimarcdn.com/ftaedge00/hmongusatv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Hmong Star TV
+https://livefta.malimarcdn.com/ftaedge00/hmongstartv.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",KhmerTV
+https://livefta.malimarcdn.com/ftaedge00/khmertv2020.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Tea TV 2
+https://livefta.malimarcdn.com/ftaedge00/teatv2.sdp/playlist.m3u8
```

Comment:
Language of this channel is not only Vietnamese, I see some English parts.
![vlcsnap-2021-05-31-17h35m40s475](https://user-images.githubusercontent.com/30985701/120215939-a65bf900-c236-11eb-9e16-e89790dac5f6.png)